validator/src/main/resources/expected-data-template/defaultExpectedMetric.mustache

```diff
@@ -11,3 +11,73 @@
     -
       name: statusCode
       value: 200
+
```

Comment:
i'm afraid changing this template would break the test in SDK repo which would probably be using this template as well with another sample app, we might either use another template or let them to use our latest sample app image.

Reply:
looped through the sdk repos, where the default template is not used, so it's not a concern.